File: src/client/xnative/lion.xdroid/Service/MobileService.cs
using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Support.Annotation;

namespace Lion.XDroid.Service
{
    [Service]
    [IntentFilter(new[] { "kr.co.odinsoftware.LionService" })]
    public class MobileService : Android.App.Service
    {
        public override void OnCreate()
        {
            base.OnCreate();
        }

        [return: GeneratedEnum]
        public override StartCommandResult OnStartCommand(Intent intent, [GeneratedEnum] StartCommandFlags flags, int startId)
        {
            return StartCommandResult.Sticky;
        }

        [Nullable()]
        public override IBinder OnBind(Intent intent)
        {
            return null;
        }
    }
}

File: src/client/xcode/Lion.Droid/app/src/main/java/com/odinsoftware/lottolion/service/MobileService.java
package com.odinsoftware.lottolion.service;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;

/**
 * Created by limyg on 2017-04-27.
 */
public class MobileService extends Service {
    @Override
    public void onCreate() {
        super.onCreate();
    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        return START_STICKY;
    }
    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

